.github/ISSUE_TEMPLATE/---feature-request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: "\U0001F389 Feature request"
+about: Suggest an idea for this project
+
+---
+
+<!--
+Please report issues regarding specific projects in their respective issue trackers, e.g.:
+ - Akka-HTTP: https://github.com/akka/akka-http/issues
+ - Alpakka:   https://github.com/akka/alpakka/issues
+ - Akka Persistence Cassandra Plugin: https://github.com/akka/akka-persistence-cassandra/issues
+ - ...
+
+Please explain your issue precisely, and if possible provide a reproducer snippet (this helps resolve issues much quicker).
```

Comment:
This sounds more relevant for issue template. Perhaps something like "Please explain your problem and use case precisely."